int16 m_pinNumber
int64 m_setpoint